plugins/chat/lib/chat/mailer.rb

```diff
@@ -6,6 +6,9 @@ def self.send_unread_mentions_summary
       return unless SiteSetting.chat_enabled
 
       users_with_unprocessed_unread_mentions.find_each do |user|
+        # Apply modifier to `true` -- this allows other plugins to block the chat summary email send
+        next if !DiscoursePluginRegistry.apply_modifier(:chat_mailer_summary_user, true, user)
```

Comment:
A more explicit name might be better? not sure what to expect from `:chat_mailer_summary_user` when reading it